docs/core-concepts/core-sdk/cli/dotnet-install-script.md

```diff
@@ -0,0 +1,119 @@
+dotnet-install scripts reference
+================================
+
+## NAME
+dotnet-install.ps1 | dotnet-install.sh - script used to install the CLI tools and shared runtime
+
+## SYNOPSIS
+dotnet-install.sh [--channel] [--version]
+    [--install-dir] [--debug] [--no-path] 
+    [--shared-runtime]
+
+dotnet-install.ps1 [-Channel] [-Version]
+    [-InstallDir] [-Debug] [-NoPath] 
+    [-SharedRuntime]
+
+## DESCRIPTION
+The `dotnet-install` scripts are used to perform a non-admin install of the CLI toolchain and the shared runtime. Their 
+main use case is to help with automation scenarios and non-admin installations. There are two scripts, one for PowerShell 
+that works on Windows and a bash script that works on Linux/OS X. They both have the same behavior. Bash script also 
+"understands" PowerShell switches so you can use them across the board. 
+
+Installation scripts will download the ZIP/tarball from the CLI build drops and will proceed to install it in either the 
+specified location or in a different location is the `--install-dir` is specified. By default, the installation script 
+will download the entire CLI and install it; if you want to get just the shared runtime, you can specify the `--shared-runtime`
+argument. 
+
+By default, the script will add the install location to the $PATH for the current session. This can be overriden if the 
+`--no-path` argument is used. 
+
+The install scripts will not install [dependencies](https://github.com/dotnet/cli/tree/rel/1.0.0/Documentation/reqs.md) 
+that are needed for each of the component. That is up to the invoker to do prior to installing bits via the script. 
```

Comment:
If you don't you will get an error. Script does not install dependencies because it would have to be sudo-ed for that and the whole point of it is to not be sudoed.